static/app/components/searchSyntax/parser.tsx

```diff
@@ -530,6 +530,7 @@ export class TokenConverter {
    * [0]:https://pegjs.org/documentation
    */
   predicateFilter = <T extends FilterType>(type: T, key: FilterMap[T]['key']) => {
+    // @ts-expect-error Unclear why this isn’t resolving correctly
```

Comment:
I think this is related to https://devblogs.microsoft.com/typescript/announcing-typescript-4-6/#indexed-access-inference-improvements, it probably does work better now :D

Reply:
@JonasBa I looked at this for a few hours, if you want to try and understand why it's not resolving correctly be my guest 😬